sechub-web-ui-solution/docker/nginx/nginx.conf

```
@@ -0,0 +1,72 @@
+worker_processes 1;
+
+events {
+  worker_connections 1024;
+}
+
+http {
+  access_log /dev/stdout;
+  error_log /dev/stderr;
+  sendfile on;
+  tcp_nopush on;
+  tcp_nodelay on;
+  keepalive_timeout 65;
+  types_hash_max_size 2048;
+
+  server {
+    listen 4443 ssl http2;
+    server_name _;
+
+    root /var/www/html;
+
+    location / {
+      # Redirect / to login page
+      rewrite ^/$ /login last;
+      # index index.html;
+    }
+
+    location /api {
+      proxy_pass https://web-server-internal:4443/api;
+      include sechub_proxy_settings.conf;
+    }
+
+    location /error {
+      proxy_pass https://web-server-internal:4443/error;
+      include sechub_proxy_settings.conf;
+    }
+
+    location /login {
+      proxy_pass https://web-server-internal:4443/login;
+      include sechub_proxy_settings.conf;
+    }
+
+    location /oauth2 {
+      proxy_pass https://web-server-internal:4443/oauth2;
+      include sechub_proxy_settings.conf;
+    }
+
+    location /health {
+      # no access logging of health checks
+      access_log off;
+    }
```

I guess the healthcheck is already working, but it could be an improvement if the return code is specified.
```suggestion
location /health {
  # no access logging of health checks
  access_log off;
  return 200 'OK';
  add_header Content-Type text/plain;
}
```

---

Good suggestion!
Is now implamented.